lib/WP_Auth0_LoginManager.php

```diff
@@ -321,8 +326,15 @@ public function implicit_login() {
 			);
 		}
 
-		// Populate legacy userinfo property.
-		$decoded_token->user_id = $decoded_token->sub;
+		// Remove unneeded ID token attributes.
+		foreach ( array( 'iss', 'aud', 'iat', 'exp', 'nonce', 'clientID' ) as $attr ) {
+			unset( $decoded_token->$attr );
+		}
+
+		// Populate legacy user_id property, if not provided.
+		if ( ! isset( $decoded_token->user_id ) ) {
+			$decoded_token->user_id = $decoded_token->sub;
```

Comment:
why? if you ensure you always have a `sub` you can make your code rely on that.

Reply:
Again, standardizing between the two. Implicit has both and "make your code rely on that" is a little more involved than it might seem.